Backend/src/test/java/idealab/api/repositories/EmployeeRepoTest.java

```diff
@@ -0,0 +1,23 @@
+package idealab.api.repositories;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import idealab.api.repositories.EmployeeRepo;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class EmployeeRepoTest {
+    @Mock
+    private EmployeeRepo employeeRepo;
+    
+    @Test
+    /**
+     * Make sure the query we created runs and that it doesn't throw
+     * an error.
+     */
+    public void makeSureFindAllEmployeeBasicsWorks() {
+        employeeRepo.findAllEmployeeBasics();
```

Comment:
What's the purpose of this test?

Reply:
I saw your comment on my branch, and just to reiterate: I think you're falling into the functional/integration testing vs the unit testing. This test isn't actually checking that the query is returning what's expected.

Reply:
I am still thinking there is value in integration tests, specifically for these queries. Thoughts?